pkg/minikube/driver/driver.go

```diff
@@ -106,6 +108,11 @@ func BareMetal(name string) bool {
 	return name == None || name == Mock
 }
 
+// NeedsRoot returns true if driver needs to run with root privileges
+func NeedsRoot(name string) bool {
+	return name == None || name == Podman
```

Comment:
This is fine for a first iteration, but do know that is possible to use podman without root:

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/building_running_and_managing_containers/index#running-rootfull-or-rootless_building-running-and-managing-containers

I'm not sure what the fastest way to notice whether or not the system configuration requires root or not, however.

Reply:
Can't wait for this!